### DISCUSSION TITLE
Security - Secure communication

### DISCUSSION THREAD

**Discussion by dmarro89** [dmarro89/dare-db#5] [Ideas]
Do you have any idea on how to ensure in an efficient way a secure communication between client/server?

**Comment by vdmitriyev** [comment]
### HTTPS Implementation

HTTPS could be implemented using standard Go libraries using [ListenAndServeTLS](https://pkg.go.dev/net/http#ListenAndServeTLS)

Materials:

* Example - https://gist.github.com/6174/9ff5063a43f0edd82c8186e417aae1dc
* [Create Secure Clients and Servers in Golang Using HTTPS](https://youngkin.github.io/post/gohttpsclientserver/)

**Reply by dmarro89** [reply]
Ok, I've created the issue -> https://github.com/dmarro89/dare-db/issues/6
If you want, you can try to implement it. Open the PR and I'll review it.
Thanks
